CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/FileLoggingRequestHandler.swift

```diff
@@ -78,6 +78,23 @@ public class FileLoggingRequestHandler {
 
         case "logging_getConfig":
             return Database.log.file.config
+            
+        case "logging_getLogsInZip":
```

Comment:
Just want to confirm that this is the endpoint name as .NET?

Reply:
as per the PR it is: "logging_getLogsInZip"

https://github.com/couchbaselabs/mobile-testkit/pull/1600